app/dashboard/courses/[courseId]/VideoList.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import { getCourseVideosById } from "@/app/api/courses/courses.api";
+import { convertSecondsToMinutes } from "@/app/configs/utils/format";
+import { useQuery } from "@tanstack/react-query";
+
+export const VideoList = ({ id }: { id: string }) => {
+  const { data: videos } = useQuery({
+    queryKey: ["videos", id],
```

Comment:
Probably it is ok whe you are working with firebase to use collection name as key, here, but you can use the same collection with similar output in different places - imagine you have 2 pages - on the first one you have list of videos and on the other one - another, similar list of videos. If you will have just few dozens of keys - it become very hard to semantically aontrol them. There is no ideal solution, but sometimes it will be easier to use routings system as base for your keys (for example `"cources/{courseId}/videos"` - using this approach you will not rewrite all you videos from previous course and site navigation become much more smooth)